From 5209f797e03203fbfd2313eecdcf79628751351f Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Fri, 27 Nov 2015 15:04:06 +0800
Subject: [PATCH] MLK-11914-1 video: mipi_dsi: correct wrong buf index usage

The 'buf' index loop is obviously wrong. So correct
it.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
(cherry picked from commit 9efddb6040f292ae08a6bd7faa5bc5182e0fa92a)

Conflicts:
	drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c
---
 drivers/video/mxc/mxcfb_hx8369_wvga.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/mxc/mxcfb_hx8369_wvga.c b/drivers/video/mxc/mxcfb_hx8369_wvga.c
index 7be7cba..0216c0b 100644
--- a/drivers/video/mxc/mxcfb_hx8369_wvga.c
+++ b/drivers/video/mxc/mxcfb_hx8369_wvga.c
@@ -285,7 +285,7 @@ int mipid_hx8369_lcd_setup(struct mipi_dsi_info *mipi_dsi)
 	buf[3] = HX8369_CMD_SETGAMMA_PARAM_4;
 	buf[4] = HX8369_CMD_SETGAMMA_PARAM_5;
 	buf[5] = HX8369_CMD_SETGAMMA_PARAM_6;
-	buf[7] = HX8369_CMD_SETGAMMA_PARAM_7;
+	buf[6] = HX8369_CMD_SETGAMMA_PARAM_7;
 	buf[7] = HX8369_CMD_SETGAMMA_PARAM_8;
 	buf[8] = HX8369_CMD_SETGAMMA_PARAM_9;
 	err = mipi_dsi->mipi_dsi_pkt_write(mipi_dsi, MIPI_DSI_GENERIC_LONG_WRITE, buf,
-- 
1.8.0

